Foundation/NSPathUtilities.swift

```diff
@@ -57,7 +57,13 @@ public func NSTemporaryDirectory() -> String {
             return tmpdir
         }
     }
-    return "/tmp/"
+    #if os(Android)
+      // Bionic uses /data/local/tmp/ as temporary directory. TMPDIR is rarely
+      // defined.
+      return "/data/local/tmp/"
+    #else
+      return "/tmp/"
+    #endif
 #endif
```

Comment:
Could you fix the indentation please, the `#if/#else/#endif` should be on the left edge, thanks

Reply:
It would be preferable, but, Foundation is so horribly inconsistent with that, that I don't think that any formatting would make this correct :-p.

Reply:
I will fix it, because I realized that I'm using 2 spaces, instead of 4.